plugins/wrangler/api_token.go

```diff
@@ -0,0 +1,51 @@
+package wrangler
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://developers.cloudflare.com/workers/wrangler/system-environment-variables/"),
+		ManagementURL: sdk.URL("https://dash.cloudflare.com/profile/api-tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.AccountID,
+				MarkdownDescription: "The account ID for the Workers related account, can be found in the Cloudflare dashboard, can usually be inferred by Wrangler.",
```

Comment:
> can usually be inferred by Wrangler

This suggest we should add `Optional: true`

Reply:
done